### ISSUE TITLE
Duplicated French translation project on Transifex

### ISSUE THREAD

**Issue by Nekzuris** [openstreetmap/iD#10441]
### Description

`fr` and `fr_FR` are the same, `fr_FR` was created because the Transifex user interface is confusing and it should be deleted.
![image](https://github.com/user-attachments/assets/a542da4d-37f5-49c9-8967-c0562f789509)

`fr_CA` is a bit different, if any Canadians wants to add their funny words.

**Comment by tyrasd** [comment]
Agree that it does not make sense to have `fr_FR`. It was probably added by accident. Before removing it on transifex, I'd need to check if there are any differences between `fr` and `fr_FR` in the translations and if yes to conflate them accordingly.

**Comment by Nekzuris** [comment]
More context to prevent this in the future:
![Image](https://github.com/user-attachments/assets/375fb39b-b0a0-4aed-b759-9a71b7870f6d)

**Comment by tyrasd** [comment]
Agree! I also normally reject request for `xx_XX` language variants (that really do come in regularly). `fr_FR` must have been accidentally accepted.

**Comment by tbertels** [comment]
Please remove `fr_FR`, it's a lesser translation with errors including translating variables and duplicate key shortcuts:

`"title":{"format":{"context":"{base} – {context}","changes":"({changements}) {base}","changes_context":"({changements}) {base} – {contexte}"}`

`"area_fill":{"wireframe":{"description":"Pas de remplissage (Filaire)","tooltip":"Activer le mode filaire facilite la vue de l'imagerie aérienne.","key":"O"}`

`"circularize":{"title":"Arrondir","description":{"single":"Arrondir cet élément.","multiple":"Arrondir ces éléments."},"key":"O","annotation":{"feature":{"one":"L'élément a été arrondi.","many":"{n} éléments ont été arrondis.","other":"{n} éléments ont été arrondis."}}`

Users who have `fr-FR` or `fr-FR fr` in their browser languages and thus [OSM languages preferences](https://www.openstreetmap.org/preferences/advanced) will use that wrong `fr_FR` version.

You can compare these two [`fr`](https://raw.githubusercontent.com/openstreetmap/iD/refs/heads/develop/dist/locales/fr.min.json) and [`fr_FR`](https://raw.githubusercontent.com/openstreetmap/iD/refs/heads/develop/dist/locales/fr-FR.min.json) translation files on https://jsondiff.com

**Comment by tyrasd** [comment]
I have deleted the duplicate `fr_FR` locale on transifex. Sorry for the delay. 🙇 

Below are the old strings of the `fr_FR` locale downloaded as a yaml dump, for reference.

[for_use_id-editor_presets_fr_FR.yml.txt](https://github.com/user-attachments/files/25973402/for_use_id-editor_presets_fr_FR.yml.txt)
[for_use_id-editor_imagery_fr_FR.yml.txt](https://github.com/user-attachments/files/25973404/for_use_id-editor_imagery_fr_FR.yml.txt)
[for_use_id-editor_core_fr_FR.yml.txt](https://github.com/user-attachments/files/25973401/for_use_id-editor_core_fr_FR.yml.txt)
[for_use_id-editor_community_fr_FR.yml.txt](https://github.com/user-attachments/files/25973403/for_use_id-editor_community_fr_FR.yml.txt)
